### PR TITLE
Update process to contribute a new dataset

### PR DESCRIPTION
**Pull request by theomeb**

### CODE REVIEW

**Comment by theomeb** [comment]
Here is a first version for a clear process on how to contribute a dataset. Feel free to add suggestions!

File: datasets/antibodybinding/description.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Embeddings:
+ - CDR1 embeddings - 1024-dim
+ - CDR2 embeddings - 1024-dim
+ - CDR3 embeddings - 1024-dim
+
+Label:
+ - binding energy
+
+### Usage
+```
```
Comment (martinp7) [inline datasets/antibodybinding/description.md]: For a better display of the snippet

```suggestion
```python
```

File: datasets/netmhcpan-4.1-train/description.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Embeddings:
+ - CLS embeddings - 1024-dim
+
+Label:
+ - target
+  - 0: non-binding (decoy)
+  - 1: binding (hit)
+
+### Usage
+```
```
Comment (martinp7) [inline datasets/netmhcpan-4.1-train/description.md]: Same here

```suggestion
```python
```

File: datasets/pathogen/description.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Embeddings:
+ - CLS embeddings - 1024-dim
+
+Label:
+ - class
+  - 0: human
+  - 1: pathogen
+
+### Usage
+```
```
Comment (martinp7) [inline datasets/pathogen/description.md]: Same here

```suggestion
```python
```

File: datasets/pfam-32.0/description.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+ - split
+
+Embeddings:
+ - MEAN embeddings - 1024-dim
+
+Label:
+ - family_id - 17929 unique classes - 13071 present in all splits.
+
+### Usage
+```
```
Comment (martinp7) [inline datasets/pfam-32.0/description.md]: Same here

```suggestion
```python
```

File: datasets/swissProt/description.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Embeddings:
+ - MEAN embeddings - 1280-dim
+ - CLS embeddings - 1280-dim
+
+Label:
+
+
+### Usage
+```
```
Comment (martinp7) [inline datasets/swissProt/description.md]: Same here

```suggestion
```python
```

File: templates/description.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+Embeddings:
+ - CLS embeddings - 1024-dim
+
+Label:
+ - class
+  - 0: human
+  - 1: pathogen
+
+### Usage
+```
```
Comment (martinp7) [inline templates/description.md]: Same here

```suggestion
```python
```

File: datasets/antibodybinding/description.md (hunk 1 above)
Comment (theomeb) [reply datasets/antibodybinding/description.md]: 👍 updated everywhere!

**Review by martinp7 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -2,24 +2,51 @@
...
+git remote add upstream https://github.com/DeepChainBio/bio-datasets.git
+```
+
+2. Set your environment and install _pre-commit hooks_ by following the [project setup section](./CONTRIBUTING.md#project-setup-nerd_face).
 
+3. Create a new branch: `git checkout -b contribute/name-of-your-dataset`.
 
-2. Then you need to add in this folder the 3 following files:
+4. Prepare your dataset. The latter must contains the 3 following files:
     - `dataset.csv` - which is your dataset (all your features + targets) without any embeddings.
```
Comment (delfosseaurelien) [inline CONTRIBUTING.md:17]: I think I understand, the new_dataset_pull_request_template.md will be created when fork is done, and explains how to upload on GCP?

**Review by delfosseaurelien — APPROVED** [review]

File: CONTRIBUTING.md (hunk 7 above)
Comment (theomeb) [reply CONTRIBUTING.md:17]: Yes exactly!

Comment (theomeb) [reply CONTRIBUTING.md:17]: new_dataset_pull_request_template.md is a template to open a pull request

**Pull request merged**